### DISCUSSION TITLE
How to disable markdown highlighting in lsp hover popups?

### DISCUSSION THREAD

**Discussion by AlxHnr** [neovim/neovim#29142] [Q&A]
Treesitter parsers are _optional_ and the docs of [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) even say it's experimental. Yet opening the lsp hover info with its default config _requires_ the markdown parser to be present:

```
Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/treesitter/language.lua:107: no parser for 'markdown' language, see :help treesitter-parsers
stack traceback:
        [C]: in function 'error'
        /usr/share/nvim/runtime/lua/vim/treesitter/language.lua:107: in function 'add'
        /usr/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:111: in function 'new'
        /usr/share/nvim/runtime/lua/vim/treesitter.lua:41: in function '_create_parser'
        /usr/share/nvim/runtime/lua/vim/treesitter.lua:108: in function 'get_parser'
        /usr/share/nvim/runtime/lua/vim/treesitter.lua:416: in function 'start'
        /usr/share/nvim/runtime/lua/vim/lsp/util.lua:1734: in function 'handler'
        /usr/share/nvim/runtime/lua/vim/lsp/client.lua:685: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

How do I disable markdown highlighting for now (or use the previous one) until this issue is resolved?

**Comment by clason** [comment]
You don't, sorry. The **bundled** tree-sitter parsers are _not_ optional. (Nvim-treesitter is a separate plugin and irrelevant here.)

And to be clear: this error means that your Neovim installation is broken (since it is missing required runtime files).

**Comment by AlxHnr** [comment]
If you are sure that the markdown parser is supposed to be bundled, I will complain in the issue tracker of my Linux distro. Thanks.

**Reply by clason** [reply]
Very sure.
